roles/contrail_master/templates/contrail-installer.j2

```diff
@@ -0,0 +1,351 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: contrail
+  namespace: kube-system
+---
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: contrail
+  namespace: kube-system
+rules:
+  - apiGroups: [""]
+    resources:
+      - pods
+      - namespaces
+    verbs:
+      - list
+      - get
+      - watch
+  - apiGroups: ["extensions"]
+    resources:
+      - networkpolicies
+    verbs:
+      - list
+      - get
+      - watch
+---
+apiVersion: v1
+kind: ClusterRoleBinding
+metadata:
+  name: contrail
+roleRef:
+  name: contrail
+subjects:
+- kind: SystemUser
+  name: kube-system:contrail
+- kind: ServiceAccount
+  name: contrail
+  namespace: kube-system
+userNames:
+  - system:serviceaccount:kube-system:contrail
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: contrail-config
+  namespace: kube-system
+data:
+  global-config: |-
+    [GLOBAL]
+    cloud_orchestrator = openshift 
+    sandesh_ssl_enable = False
+    enable_config_service = True
+    enable_control_service = True
+    enable_webui_service = True
+    introspect_ssl_enable = False
+    config_nodes = {{ groups.oo_first_master.0 }} 
+    controller_nodes = {{ groups.oo_first_master.0 }}
+    analytics_nodes = {{ groups.oo_first_master.0 }}
+    analyticsdb_nodes = {{ groups.oo_first_master.0 }}
+    analyticsdb_minimum_diskgb = {{ analyticsdb_min_diskgb }}
```

Comment:
There are various vars in use in the role and this template which should be documented in a README.md for the role.